onadata/apps/api/permissions.py

```diff
@@ -157,6 +158,19 @@ def has_permission(self, request, view):
         if request.user.is_anonymous() and view.action == 'create':
             return True
 
+        if view.action in ['send_verification_email', 'verify_email']:
+            enable_email_verification = getattr(
+                settings, 'ENABLE_EMAIL_VERIFICATION', False
+            )
+            if enable_email_verification is None or\
+                    not enable_email_verification:
+                return False
+
+            if view.action == 'send_verification_email':
+                if request.user.username == request.data.get('username'):
```

Comment:
the equality returns a boolean, why not return that instead of writing your own returns w/bools?

Reply:
That makes sense. I'll change it.